js/lbank.js

```diff
@@ -501,8 +501,22 @@ module.exports = class lbank extends Exchange {
             let query = this.keysort (this.extend ({
                 'api_key': this.apiKey,
             }, params));
-            let queryString = this.rawencode (query) + '&secret_key=' + this.secret;
-            query['sign'] = this.hash (this.encode (queryString)).toUpperCase ();
+            let queryString = this.rawencode (query);
+            let message = this.hash (this.encode (queryString)).toUpperCase ();
+            let secretArr = ['-----BEGIN RSA PRIVATE KEY-----'];
+            let secretLength = true ? this.secret.length : null;  // eslint-disable-line
```

Comment:
let me know if there is a more standard way to get `strlen` in php.

Reply:
Yes, there is (the above line won't work) https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#working-with-array-lengths

Reply:
Pardon, ignore my previous comment, your code is fine.